### DISCUSSION TITLE
Allow for exclusion of folders for indexing

### DISCUSSION THREAD

**Discussion by aagha** [logancyang/obsidian-copilot#337] [Ideas]
I'd like to specify folders I'd like to exclude from indexing, for example folders where all my Attchments are kept. It'd be nice to be able to specify (multiple) folders we can keep from being indexed.

**Comment by JoseEchave** [comment]
+1

**Reply by logancyang** [reply]
@JoseEchave this is already live in v2.5.2, try it out and let me know if it works for you! Note that you need to force re-index the vault, not just refresh index.

**Reply by JoseEchave** [reply]
Great. Is this done then by choosing a given folder in the file explorer in obsidian and then running `Copilot: Force re-index vault for QA` ? 

Will then refresh (for new/modified files) then work only for the folders where re-index was forced?

**Reply by logancyang** [reply]
@JoseEchave You can set the exclude filter by the new command `Set exclusion for Vault QA mode`. The paths and titles will be there in the same field next time you use that command.

Yes, refresh / force re-index should always respect the exclusion.
